empress/support_files/templates/side-panel.html

```diff
@@ -221,6 +221,10 @@
       </p>
     </div>
     <div id="barplot-border-content" class="hidden">
+      <p>
+        <label for="barplot-custom-gap">Separation length</label>
```

Comment:
Only thing I can come up with is to make this a bit more verbose.

```suggestion
        <label for="barplot-custom-gap">Distance between tree and barplots</label>
```